### PR TITLE
Updated docs for usbipd-win updates

### PR DESCRIPTION
**Pull request by craigloewen-msft**
Updated the docs to reflect the changes in [usbipd-win 4.0.0](https://github.com/dorssel/usbipd-win/releases/tag/v4.0.0)

See the full changes list and discussion [here](https://github.com/dorssel/usbipd-win/discussions/774).

### CODE REVIEW

**Review by dorssel — CHANGES_REQUESTED** [review]
Overall, the changes are correct, but more changes are required. See https://github.com/dorssel/usbipd-win/wiki/WSL-support (first part, up to `udev`) for additional details.

**Review by mattwojo — APPROVED** [review]

**Comment by mattwojo** [comment]
@craigloewen-msft Looks good to me. Please have a look and merge if it seems ready.

File: WSL/connect-usb.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Setting up the USB/IP project on your Windows machine will enable common develop
 
 - Running Windows 11 (Build 22000 or later). *(Windows 10 support is possible, see note below).*
 - A machine with an x64/x86 processor is required. (Arm64 is currently not supported with usbipd-win).
```
Comment (dorssel) [inline WSL/connect-usb.md:17]:
```suggestion
- A machine with an x64 processor is required. (x86 and Arm64 are currently not supported with usbipd-win).
```

File: WSL/connect-usb.md [hunk 2]
```diff
@@ -82,7 +72,7 @@ Before attaching your USB device, ensure that a WSL command line is open.  This
 4. Once you are done using the device in WSL, you can either physically disconnect the USB device or run this command from PowerShell in *administrator* mode:
```
[the remark this answers is not in the document]
Comment (dorssel) [reply WSL/connect-usb.md:78]:
```suggestion
4. Once you are done using the device in WSL, you can either physically disconnect the USB device or run this command from PowerShell:
```

**Review by dorssel — CHANGES_REQUESTED** [review]
It is not possible to add suggestions for collapsed (non-changed) lines, but line 37 should become:

```
> Alternatively, you can also install the usbipd-win project using [Windows Package Manager](/windows/package-manager/winget/) (winget). If you have already installed winget, just use the command: `winget install --exact dorssel.usbipd-win` to install usbipd-win.
```

(the part about automatic reboots is obsolete).

**Pull request merged**